storage/ResidentDatabaseSchema.ts

```diff
@@ -15,7 +15,10 @@ type ResidentDatabaseSchema = NamedSchema<
       value: {
         type: string;
         images: string[];
-        notes: string;
+        notes: {
+          value: string;
+          isPostVisitAction: boolean;
+        };
```

Comment:
These could probably do with sharing the `Note` interface. That might mean pulling it out into a `DatabaseSchema.d.ts` file for schema types to avoid a circular reference.